ouroboros-consensus-cardano-test/src/Test/Consensus/Cardano/MockCrypto.hs

```diff
@@ -38,6 +39,8 @@ instance Crypto MockCryptoCompatByron where
   type DSIGN    MockCryptoCompatByron = Ed25519DSIGN
   type HASH     MockCryptoCompatByron = Blake2b_256
   type KES      MockCryptoCompatByron = MockKES 10
-  type VRF      MockCryptoCompatByron = FakeVRF
+  type VRF      MockCryptoCompatByron = MockVRF
```

Comment:
This commit is marked WIP, so should I not ask why `FakeVRF -> MockVRF`?

Reply:
FakeVRF imposes some additional requirements which are not satisfied by `UnifiedVRF` and are hard to make so. Since we don't seem to actually use the fake functionality here, `Mock` seemed better.